### DISCUSSION TITLE
Relecture du rapport de stage

### DISCUSSION THREAD

**Discussion by IsoerNC** [adriens/domaine-nc-javafx#37] [Q&A]
Nous avons rédigé notre rapport de stage, il n'est pas encore entièrement terminer mais nous aimerions avoir votre retour. Comment devrions nous vous l'envoyer ? Est ce qu'on l'envois a l'adresse écrite dans le google drive fourni par Mme Selmaoui ou bien vous préférez ? @adriens

**Comment by adriens** [comment]
hello @IsoerNC , où réigez-vous le rapport ? Sur docs ou ailleurs ?

**Reply by IsoerNC** [reply]
Oui c'est sur un document Word
